Scripts/index.js

```diff
@@ -0,0 +1,77 @@
+import { register, login } from "./auth.js";
+
+// Registration and login functions
+
+document.addEventListener("DOMContentLoaded", () => {
+  const registerForm = document.getElementById("registerForm");
+  const loginForm = document.getElementById("login-form");
+
+  registerForm.addEventListener("submit", handleRegister);
+  loginForm.addEventListener("submit", handleLogin);
+});
+
+/**
+ * Handles the registration process.
+ * @param {Event} event - The submit event from the registration form.
+ */
+async function handleRegister(event) {
+  event.preventDefault();
+  const name = document.getElementById("regName").value;
+  const email = document.getElementById("regEmail").value;
+  const password = document.getElementById("regPassword").value;
+
+  try {
+    const response = await register(name, email, password);
+    console.log("Registration successful", response);
+
+    // If response includes the jwtToken
+    localStorage.setItem("jwtToken", response.accessToken);
+
+    // Redirect to the feed page
+    redirectUser("/feed");
+  } catch (error) {
+    console.error("Registration failed", error);
+    displayErrorMessage("registerError", error.message);
+  }
+}
+
+/**
+ * Handles the login process.
+ * @param {Event} event - The submit event from the login form.
+ */
+async function handleLogin(event) {
+  event.preventDefault();
+  const email = document.getElementById("login-email").value;
+  const password = document.getElementById("login-password").value;
+
+  try {
+    const response = await login(email, password);
+    console.log("Login successful", response);
+    localStorage.setItem("jwtToken", response.accessToken);
+    redirectUser("/feed");
+  } catch (error) {
+    console.error(error);
+    displayErrorMessage("loginError", error.message);
+  }
+}
+
+/**
+ * Redirects the user to the specified page.
+ * @param {string} page
+ */
+function redirectUser(page) {
+  window.location.href = page;
+}
+
```

Comment:
Functions like redirectUser and displayErrorMessage would be better placed in a helpers or utils folder. They could be used anywhere on the site.